features/step_definitions/shout_steps.js

```diff
@@ -6,29 +6,30 @@ var {defineSupportCode} = require('cucumber');
 
 defineSupportCode(function({Given, When, Then}) {
   const ARBITARY_MESSAGE = 'Hello, world'
+  var shouty = new Shouty()
```

Comment:
This will cause state to leak between scenarios. A new instance needs to be made for each scenario. Either in the World (which you removed), or in a `Before` hook (which I think is better).